src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -825,6 +825,30 @@ internal static void ValidateSpecifiedAuthentication(PSCredential credential, st
 
         #endregion
 
+        /// <summary>
+        /// Parse a hostname used with SSH Transport to get embedded
+        /// username and/or port.
+        /// </summary>
+        /// <param name="hostname">host name to parse</param>
+        /// <param name="host">resolved target host</param>
+        /// <param name="userName">resolved target user name</param>
+        /// <param name="port">resolved target port</param>
+        protected void ParseSshHostName(string hostname, out string host, out string userName, out int port)
```

Comment:
This can be a static method.

Reply:
Need access to `this` so can't be static